modules/cnf-installing-the-operators.adoc

```diff
@@ -0,0 +1,273 @@
+// CNF-950 4.7 Installing the operators
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-installing-the-operators_{context}"]
+= Installing the operators
+
+[id="cnf-installing-the-performnce-addon-operator_{context}"]
+== Installing the Performance Addon Operator
+
+Install the Performance Addon Operator using the {product-title} CLI or the web console.
+
+.Procedure
+
+. Create the `operator-namespace.yaml` file:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    openshift.io/cluster-monitoring: "true"
+  name: openshift-performance-addon-operator
+spec: {}
+----
+
+. Create the `operator_catalogsource.yaml` file:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: performance-addon-operator
+  namespace: openshift-marketplace
+spec:
+  displayName: Openshift Performance Addon Operator
+  icon:
+    base64data: ""
+    mediatype: ""
+  image: quay.io/openshift-kni/performance-addon-operator-index:4.7-snapshot
```

Comment:
Are you intentionally using the upstream PAO snapshot here?

Reply:
@vitus133 - should we move this?

Reply:
@StephenJamesSmith Could you please remove the entire block in lines 29...47?
Since we are using the official catalog, CatalogSource resource is not required

Reply:
Removed.